doc/t265.md

```diff
@@ -202,3 +202,107 @@ Below is a sample calibration file with inline comments (not a valid file format
         }
     ]
 }
+```
+### Extrinsic Calibration for Wheeled Odometry Examples
+
+ - All calibration metrics are relative to T265 origin frame. I.e.: They are offsets/rotations *from* the T265 origin *to* the robot's origin. Said another way, we transform a point from frame B (Robot Odometry) to frame A (T265 Pose.)
```

Comment:
possibly even more clear by adding a formula, A_p = H_AB * B_p, where A_p is the point expressed in frame A, B_p is the point expressed in frame B, and H_AB is the corresponding homogeneous transformation  (http://paulfurgale.info/news/2014/6/9/representing-robot-pose-the-good-the-bad-and-the-ugly)

Reply:
This makes sense. So just to be clear...H_AB would effectively be the transformation matrix derived from the values in the JSON calibration file. Is that correct?

Reply:
yes, @krazycoder2k